### PR TITLE
LEAN-Drug-Reagent

### PR DESCRIPTION
**Pull request by Conchuckter**
[FEATURE]

## About the PR 

**This PR adds lean as a reagent.**

Its effects include: 

Making you less jittery.
Curing the common cold and both types of flu
Addiction

OD can cause:

Drowsiness
Fainting
Losebreath
Toxin damage

Recipe: Robustissin + Ethanol + Lemonade or Cola

## Why's this needed? 
Someone on discord requested it and seeing as drugs are one area of chemistry we are lacking in it would be good to have a few more.

Easy to make gimmick drugs are good for RP.

## Changelog

```changelog
(u)Conchuckter
(+)Added Lean as a new reagent
(+)Made haloperidol remove Lean
```

### CODE REVIEW

**Comment by DisturbHerb** [comment]
oh god why

**Comment by Conchuckter** [comment]
Cause lean?

**Comment by colossusqw** [comment]
yes more drugs

File: code/modules/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -4212,3 +4212,75 @@ datum
...
+				..()
+				return
+
+			do_overdose(var/severity, var/mob/M, var/mult = 1)
+				var/effect = ..(severity, M)
+				M.stuttering += rand(0,2)
+				if(severity == 1)
+					if(effect <= 4)
+						M.emote(pick("drool", "shiver", "shudder", "blink"))
+						M.visible_message("<span class='alert'><b>[M.name] calapses!</b> ")
```
Comment (NightmareChamillian) [inline code/modules/chemistry/Reagents-FoodDrink.dm:4254]: I think you meant collapses?

Comment (Conchuckter) [reply code/modules/chemistry/Reagents-FoodDrink.dm:4254]: Whoops, I'll change it when I have the chance

**Comment by wolfolotl** [comment]
seems like it'd open up a can of worms culturally, and also you can already get high on ingame cough syrup

**Comment by Conchuckter** [comment]
> 

I wasn't referencing anything when I made this reagent and I didn't even know much about it before making it. While you can get high off cough syrup it requires you to od which comes with alot of downsides.

**Comment by Cboyjet4** [comment]
I LOVE LEAN

**Comment by Tarmunora** [comment]
gonna go with no on this

**Comment by Conchuckter** [comment]
> gonna go with no on this

Can you elaborate on why?

**Comment by flappybatpal** [comment]
Not a dev but presumably primarily it's because it's a meme that may not last and the risk is filling the game with references that are dated in five years.

**Pull request closed without merging**